frontend/src/Dashboard.tsx

```diff
@@ -161,6 +161,10 @@ const Dashboard = () => {
     return petNameIncludes;
   });
 
+  const formatDate = (date: Date) => {
```

Comment:
Ideally, place this function in the `utils` file so that it can be reused and not duplicated like you've done in the PR.